src/components/molecules/Cv/Cv.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import {
+  AboutMe,
+  BasicData,
+  CreateDescription,
+  Languages,
+  PersonalCard,
+  Skills,
+} from './components';
+import { CvDescription, CvHeading } from '../../atoms';
+
+export const Cv: TCv = () => <></>;
```

Comment:
No to powinno przynajmniej zwracać `children`, abyśmy wrapowali całość, nie jak obecnie w `Box`.

```
<Cv>
  <Cv.AboutMe />
</Cv>

```